dis_snek/models/discord_objects/emoji.py

```diff
@@ -35,7 +35,7 @@ def unicode(cls, emoji: str):
         return cls(name=emoji)
 
     def __str__(self) -> str:
-        return f"<{'a:' if self.animated else ''}{self.name}:{self.id}>"  # <:thinksmart:623335224318754826>
+        return f"<{'a:' if self.animated else ':'}{self.name}:{self.id}>"  # <:thinksmart:623335224318754826>
```

Comment:
I like how you kept my random comment these ;p

Reply:
I saw no reason to remove it 😛